tests/components/litejet/test_config_flow.py

```diff
@@ -76,3 +76,22 @@ async def test_import_step(hass):
     assert result["type"] == "create_entry"
     assert result["title"] == test_data[CONF_PORT]
     assert result["data"] == test_data
+
+
+async def test_options(hass):
+    """Test updating options."""
+    entry = MockConfigEntry(domain=DOMAIN, data={CONF_PORT: "/dev/test"})
+    entry.add_to_hass(hass)
+
+    result = await hass.config_entries.options.async_init(entry.entry_id)
```

Comment:
Please set up the integration before starting the options flow to make sure that the config flow module is loaded.